vars/main.yml

```diff
@@ -4,3 +4,4 @@ dockercheck_svc_group: "root"
 dockercheck_path: "/usr/local/bin"
 dockercheck_metric_name: "container_status"
 dockercheck_label: "com.docker.compose.project"
+disk_usage_threshold: -1
```

Comment:
shouldn't be 90?

Reply:
no, I check that to [set](https://github.com/StuartApp/dockercheck/pull/5) as -1 in script as deafult, so will set it 90 for the project that we want it to be checked